plugins/synapticsmst/fu-plugin-synapticsmst.c

```diff
@@ -39,18 +37,108 @@ synapticsmst_common_check_supported_system (FuPlugin *plugin, GError **error)
 	return TRUE;
 }
 
+/* creates MST-$str-$BOARDID */
+static void
+fu_plugin_synapticsmst_create_simple_guid (FuDevice *fu_device,
+					   SynapticsMSTDevice *device,
+					   const gchar *str)
+{
+	guint32 board_id = synapticsmst_device_get_board_id (device);
+	g_autofree gchar *guid = g_strdup_printf ("MST-%s-%u", str, board_id);
```

Comment:
maybe using `%04x` as a format type might be better for a guint32 number?

Reply:
Mostly limited by already built CABs (and build scripts/tools) that use `%u` however.

Reply:
Aha, okay, ignore me.